helm-charts/orchestrator/templates/sonataflows.yaml

```diff
@@ -23,6 +23,8 @@ metadata:
   annotations:
     "meta.helm.sh/release-name": {{ .Release.Name}}
 spec:
+  monitoring:
+    enabled: {{ .Values.orchestrator.sonataflowPlatform.monitoring.enabled }}
```

Comment:
Might be missing the addition of this new fields in the chart README file here: https://github.com/rhdhorchestrator/orchestrator-helm-operator/blob/main/helm-charts/orchestrator/README.md